### DISCUSSION TITLE
MIME Types

### DISCUSSION THREAD

**Discussion by MalteKiefer** [immich-app/immich#425] [General]
I just need to know, which MIME types we should add?
I will later add this to the server and test it.

Currently, to do:

- tif(f) #406 
- avif #407

@alextran1502 can you let me know where I have to change it all? I know on the CLI where, but server and mobile version?

**Comment by alextran1502** [comment]
You can add more in here 

server/apps/immich/src/config/asset-upload.config.ts

**Reply by MalteKiefer** [reply]
Why are here more than in the CLI version?
